src/test/kotlin/apply/application/MailTargetServiceTest.kt

```diff
@@ -12,119 +11,148 @@ import apply.domain.evaluationtarget.EvaluationStatus.WAITING
 import apply.domain.evaluationtarget.EvaluationTargetRepository
 import apply.domain.user.UserRepository
 import apply.domain.user.findAllByEmailIn
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.collections.shouldBeEmpty
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.shouldBe
+import io.mockk.clearAllMocks
 import io.mockk.every
-import io.mockk.impl.annotations.MockK
+import io.mockk.mockk
 import io.mockk.verify
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.Test
-import support.test.UnitTest
+import support.test.spec.afterRootTest
 
-@UnitTest
-class MailTargetServiceTest {
-    @MockK
-    private lateinit var evaluationTargetRepository: EvaluationTargetRepository
+class MailTargetServiceTest : BehaviorSpec({
+    val evaluationTargetRepository = mockk<EvaluationTargetRepository>()
+    val userRepository = mockk<UserRepository>()
 
-    @MockK
-    private lateinit var userRepository: UserRepository
+    val mailTargetService = MailTargetService(evaluationTargetRepository, userRepository)
 
-    private lateinit var mailTargetService: MailTargetService
+    Given("특정 평가에 대한 평가 대상자가 있는 경우") {
+        val evaluationId = 1L
 
-    @BeforeEach
-    internal fun setUp() {
-        mailTargetService = MailTargetService(evaluationTargetRepository, userRepository)
-    }
-
-    @Test
-    fun `평가 상태에 따라 (null, 전체) 메일 발송 대상들의 이메일 정보를 불러온다`() {
         every { evaluationTargetRepository.findAllByEvaluationId(any()) } returns listOf(
-            createEvaluationTarget(userId = 1L, evaluationStatus = WAITING),
-            createEvaluationTarget(userId = 2L, evaluationStatus = PENDING),
-            createEvaluationTarget(userId = 3L, evaluationStatus = PASS),
-            createEvaluationTarget(userId = 4L, evaluationStatus = FAIL)
+            createEvaluationTarget(evaluationId = evaluationId, userId = 1L, evaluationStatus = WAITING),
+            createEvaluationTarget(evaluationId = evaluationId, userId = 2L, evaluationStatus = PENDING),
+            createEvaluationTarget(evaluationId = evaluationId, userId = 3L, evaluationStatus = PASS),
+            createEvaluationTarget(evaluationId = evaluationId, userId = 4L, evaluationStatus = FAIL)
         )
         every { userRepository.findAllById(any()) } returns listOf(
             createUser(id = 1L, email = "waiting@email.com"),
             createUser(id = 2L, email = "pending@email.com"),
             createUser(id = 3L, email = "pass@email.com"),
             createUser(id = 4L, email = "fail@email.com")
         )
-        val actual = mailTargetService.findMailTargets(EVALUATION_ID)
-        assertThat(actual).hasSize(4)
+
+        When("해당 평가의 모든 평가 대상자에 대한 이메일 정보를 조회하면") {
+            val actual = mailTargetService.findMailTargets(evaluationId)
+
+            Then("모든 평가 대상자의 이메일 정보를 확인할 수 있다") {
+                actual shouldHaveSize 4
```

Comment:
r: `이메일 정보를 확인할 수 있다`인데 사이즈로 판별하고 있네요!
```suggestion
            Then("모든 평가 대상자의 이메일 정보를 확인할 수 있다") {
                actual shouldHaveSize 4
```